libs/community/langchain_community/llms/azureml_endpoint.py

```diff
@@ -53,12 +53,13 @@ def call(
 
 
 class AzureMLEndpointApiType(str, Enum):
-    """Azure ML endpoints API types. Use `realtime` for models deployed in hosted
+    """Azure ML endpoints API types. Use `dedicated` for models deployed in hosted
     infrastructure, or `serverless` for models deployed as a service with a
     pay-as-you-go billing or PTU.
     """
```

Comment:
"Use `dedicated` for models deployed in hosted infrastructure (also known as Online Endpoints in Azure Machine Learning)."

Reply:
Fixed